api/RTCCertificate.json

```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
```

Comment:
This file shouldn't be there.

You need either to rebase your branch to master (your local master), or create a new branch out of your local master.